src/beacon/app/dashboard/analytics/DistrictAnalysisPage.tsx

```
@@ -0,0 +1,978 @@
+        `${config.apiUrl}/network-analysis/districts/${encodeURIComponent(district)}?country=${encodeURIComponent(country)}`
+      )
+      
+      // Check if this call was cancelled
+      if (callRef.cancelled) return
+
+      if (response.ok) {
+        const districtData = await response.json()
+        
+        if (callRef.cancelled) return
+
+        if (districtData.data && districtData.data.devicesList) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)